src/goose/brokers/redis/commands.clj

```diff
@@ -155,6 +155,22 @@
 (defn del-from-list [conn list element]
   (wcar* conn (car/lrem list 1 element)))
 
+(defn list-position-multiple [conn list elements]
+  (wcar* conn (doseq [e elements]
+                (car/lpos list e))))
+
+(defn del-from-list-multiple [conn list elements]
+  (wcar* conn (doseq [e elements]
+                (car/lrem list 1 e))))
+
+(defn del-from-list-and-enqueue-front-multiple [conn list elements]
+  (wcar* conn (doseq [e elements]
+                (car/atomic
+                       conn atomic-lock-attempts
+                       (car/multi)
+                       (car/lrem list 1 e)
+                       (car/rpush list e)))))
```

Comment:
question: Since this will be run for multiple elements, does the `atomic-lock-attempts` defined earlier is still big enough number ?